util-count.xsl

```diff
@@ -13,7 +13,7 @@
 				<xsl:value-of select="count($items)" />
 			</xsl:when>
 			<xsl:otherwise>
-				0
+				<xsl:text>0</xsl:text>
```

Comment:
Why the change for text here?

Reply:
I usually try not to leave "free" text in templates. 
They tend to fuck up the indentation in the dom.

Reply:
Yeah I meant compared to keeping it a number.. I worry about our comparisons: If we take this result (string '0') and test "if greater than 0 (number)", what will happen?

Could you use value-of instead?

Reply:
@hotdoy Ping! What do you think?

Reply:
I don't see why value-of a string would be better than a text but let's test it and see